mentat/llm_api_handler.py

```diff
@@ -356,13 +383,14 @@ async def call_llm_api(
         return response
 
     @api_guard
-    async def call_embedding_api(
+    def call_embedding_api(
```

Comment:
hmm a little too bad Chroma doesn't support async. It's probably not an issue now but in the future if were doing embeddings in the background it could make mentat unresponsive

Reply:
Look like some other people on Discord also want async, but not dice yet: https://discord.com/channels/1073293645303795742/1073293646083919946/1191885640120414218